app/_hub/kong-inc/request-termination/index.md

```diff
@@ -69,16 +73,31 @@ params:
       default: "`application/json; charset=utf-8`"
       datatype: string
       description: Content type of the raw response configured with `config.body`.
+    - name: trigger
+      required: false
+      default:
+      description: When not set, the plugin always activates. When set to a string, the plugin will activate exclusively on requests containing either a header or a query parameter named like said string.
+    - name: echo
+      required: false
+      default: false
+      description: When set, the plugin will echo a copy of the request back to the client. The main usecase for this is debugging. It can be combined with `trigger` in order to debug requests on live systems without disturbing real traffic
```

Comment:
```suggestion
      description: When set, the plugin will echo a copy of the request back to the client. The main usecase for this is debugging. It can be combined with `trigger` in order to debug requests on live systems without disturbing real traffic.
```